whois-api/src/main/java/net/ripe/db/whois/api/elasticsearch/ElasticFullTextIndex.java

```diff
@@ -120,6 +120,15 @@ protected void update() throws IOException {
         elasticIndexService.updateMetadata(new ElasticIndexMetadata(end, source));
     }
 
+    private SerialEntry getSerialEntry(int serial) {
+        try {
+            return JdbcRpslObjectOperations.getSerialEntry(jdbcTemplate, serial);
+        } catch (Exception e) {
+            LOGGER.debug("SerialDao.getById({})", serial, e);
```

Comment:
Can you make the message a bit more descriptive, such as `"Caught exception reading serial {} from the database", serial, e`

Reply:
Also add `, ignoring` to be clear we didn't do anything about it